examples/README.md

```diff
@@ -9,6 +9,10 @@ Various examples that depict MLflow tracking, project, and serving use cases.
 * `h2o` depicts how MLflow can be use to track various random forest architectures to train models 
 for predicting wine quality.
 * `hyperparam`  shows how to do hyperparameter tuning with MLflow and some popular optimization libraries.
+* `keras` 
+modifies [a Keras classification example](https://github.com/keras-team/keras/blob/master/examples/reuters_mlp.py)
```

Comment:
Maybe permalink to the specific commit? Fun fact you can press "y" while viewing a file on GitHub to get a permalink, e.g. https://github.com/keras-team/keras/blob/ed07472bc5fc985982db355135d37059a1f887a9/examples/reuters_mlp.py